packages/zowe-explorer-ftp-extension/package.json

```diff
@@ -54,8 +54,8 @@
   "devDependencies": {
     "@types/tmp": "0.2.0",
     "concurrently": "^5.2.0",
-    "webpack": "^4.42.1",
-    "webpack-cli": "^3.3.11"
+    "webpack": "^5.65.0",
```

Comment:
It seems that `--info-verbosity` is no longer an option after `webpack@5`
Source: https://github.com/microsoft/vscode-generator-code/issues/231

The other issue being the `--watch` mode for webpack preventing the VSCode Debugger from launching the new Window with the built changes.
Source: Personal testing 😋

Reply:
It looks like the webpack update also broke the theia tests since theia is still on `webpack@4` too, reverted & removed 17 CI checks since doesn't look to be supported by `webpack@4`